### PR TITLE
Healthy Culture of Controversy

### PR DESCRIPTION
**Pull request by maktwin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ We are individually responsible for our parts of the products. Also, collectivel
 Each one of us is the owner of their code and/or projects. If something fails, they have to take care of the failure, ensure completeness and ensure the return to full health.
 
 ## Healthy Culture of Controversy
-It is ok to disagree in a discussion. A healthy culture of controversy includes that everyone's views are equally valid and valuable. Although the person who works on the task gets to have the absolute right to stop the argument at any time.
+It is ok to disagree in a discussion. A healthy culture of controversy includes that everyone's subjective views are equally valid and valuable. When it comes to objective facts, standards, references, and pure logic is the way to resolve them. 
+If there is no agreement in the discussion then escalate it to the person that is accountable or the expert of this field to make a decision.
```
Comment (isabel22) [inline README.md:24]:
```suggestion
If there is no agreement in the discussion then escalate it to the accountable person or the expert of this field to decide.
```

File: README.md [hunk 2]
```diff
@@ -20,7 +20,9 @@ We are individually responsible for our parts of the products. Also, collectivel
...
+In the discussion we need to make sure that every argument has been heard and there is no winner or loser in a discussion. A discussion shouldn’t be about ego.
```
Comment (isabel22) [inline README.md:25]:
```suggestion
In the discussion we need to make sure that every argument has been heard and there is no winner or loser. A discussion shouldn't be about ego.
```

File: README.md [hunk 3]
```diff
@@ -20,7 +20,9 @@ We are individually responsible for our parts of the products. Also, collectivel
 Each one of us is the owner of their code and/or projects. If something fails, they have to take care of the failure, ensure completeness and ensure the return to full health.
 
 ## Healthy Culture of Controversy
-It is ok to disagree in a discussion. A healthy culture of controversy includes that everyone's views are equally valid and valuable. Although the person who works on the task gets to have the absolute right to stop the argument at any time.
+It is ok to disagree in a discussion. A healthy culture of controversy includes that everyone's subjective views are equally valid and valuable. When it comes to objective facts, standards, references, and pure logic is the way to resolve them.
```
Comment (isabel22) [inline README.md:23]: Can we add an extra line after line 23?

**Review by isabel22 — COMMENTED** [review]
I left a couple suggestions. Besides that, it looks good to me 👍🏼

**Review by ant31 — APPROVED** [review]

**Pull request merged**